x/pool-incentives/keeper/keeper.go

```diff
@@ -196,6 +196,19 @@ func (k Keeper) GetLockableDurations(ctx sdk.Context) []time.Duration {
 	return info.LockableDurations
 }
 
+func (k Keeper) GetLongestLockableDuration(ctx sdk.Context) time.Duration {
+	lockableDurations := k.GetLockableDurations(ctx)
+	longestDuration := time.Duration(0)
+
+	for _, duration := range lockableDurations {
+		if duration > longestDuration {
+			longestDuration = duration
+		}
+	}
+
+	return longestDuration
+}
```

Comment:
Let's return error if the `locableDurations` are empty please and cover it with a test.

Otherwise, we risk introducing unexpected behavior around "0" if the param ever gets changed for some reason

Reply:
addressed [f5e53c7](https://github.com/osmosis-labs/osmosis/pull/4970/commits/f5e53c7523f185f4dd6fe67eb735df475519b76b)